### PR TITLE
helios: set SUPPRESS_GCLOUD_CREDS_WARNING environment variable

### PR DESCRIPTION
**Pull request by mattnworb**
Have to stop using [write_jar_script][0] as it does not support passing
in environment variables.

Writing the script inline instead, which I picked up from the scio
formula.

[0]: https://www.rubydoc.info/github/Homebrew/brew/Pathname:write_jar_script

### CODE REVIEW

**Comment by mattnworb** [comment]
before merging, I need to doublecheck how our @spotify-helios-ci-agent robot bumps the version number in each release to make sure I didn't break anything it expects to be there.

File: helios.rb [hunk 1]
```diff
@@ -9,8 +9,13 @@ class Helios < Formula
...
 
   def install
-    libexec.install "helios-tools-0.9.230-shaded.jar"
-    bin.write_jar_script libexec/"helios-tools-0.9.230-shaded.jar", "helios", "-XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none"
+    jarfile = "helios-tools-#{version}-shaded.jar"
+    libexec.install jarfile
+    (bin+"helios").write <<~EOS
+      #!/usr/bin/env bash
+      export SUPPRESS_GCLOUD_CREDS_WARNING=true
+      exec java -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none -jar #{libexec}/#{jarfile} "$@"
```
Comment (caipre) [inline helios.rb]: Is `libexec` defined somewhere? Previously it was a literal string.

**Comment by mattnworb** [comment]
confirmed that our internal release script just does `sed -i "s/$old_version/$RELEASE_TAG/g" $helios_formula_path`, so no problems here

File: helios.rb (hunk 1 above)
Comment (mattnworb) [reply helios.rb]: https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md#variables-for-directory-locations

that doc in general was really helpful to understand this formula better

**Comment by mattnworb** [comment]
apparently the environment variable I am setting in this change is brand new, and not yet in a gcloud oauth2 library release https://github.com/googleapis/google-auth-library-java/pull/207

We'll have to update the version of the library used by helios for this to be picked up.

**Pull request merged**